src/index/diskann.cpp

```diff
@@ -226,6 +226,8 @@ DiskANN::build(const DatasetPtr& base) {
         auto vectors = base->GetFloat32Vectors();
         auto ids = base->GetIds();
         auto data_num = base->GetNumElements();
+        CHECK_ARGUMENT(data_num >= 2,
+                       fmt::format("base.num_elements({}) must be greater than 2", data_num));
```

Comment:
is it related to any bug or issue?

Reply:
It is not relate any bugs. We have constrained this matter in functests, but we haven't done any checks in the code, which is not acceptable.

Reply:
should we add some unit tests or func tests to cover this new behavior?

Reply:
I have added the related unit test.

Reply:
@inabao I saw that the minimum number of elements is changed to 1. What is the consideration?

Reply:
`>1` == `>=2`, so we should keep the minimum number of elements equal to 2 and greater than 1